.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .DS_Store
 docs/_book
+.vscode/
```

Comment:
This should not be a part of project's `.gitignore` because it has nothing to do with actual project. If vscode is your main code editor you should add it to your global `.gitignore` instead.